README.md

```diff
@@ -68,15 +71,18 @@ If not, see <https://opensource.org/licenses/BSD-3-Clause>.
 # USAGE
 
 The usage of SMAC v3 is mainly the same as provided with [SMAC v2.08](http://www.cs.ubc.ca/labs/beta/Projects/SMAC/v2.08.00/manual.pdf).
-It supports the same parameter configuration space syntax and interface to target algorithms.
-Please note that we do not support the extended parameter configuration syntax introduced in SMACv2.10.
+It supports the same parameter configuration space syntax and interface to
+target algorithms. Please note that we do not support the extended parameter
+configuration syntax introduced in SMACv2.10.
```

Comment:
Since you touched this:
That's not true anymore, right? 
Except the extended forbidden constraints, SMAC supports the extended pcs format.

Reply:
> I think we support the ACLIB format?

Yes, we do.
However, SMAC2 supports a bit more than ACLib, i.e., arbitrary forbidden constraints.

Reply:
If you suggest a different wording I will change this. Otherwise, I'd appreciate if you accept this ;)